views/js/qtiCreator/widgets/interactions/graphicInteraction/Widget.js

```diff
@@ -60,7 +60,7 @@ define([
                     responsive  : $container.hasClass('responsive'),
                     resize      : function() {
                         if(typeof resize === 'function') {
-                            resize.apply(this, arguments);
+                            resize.bind(arguments);
```

Comment:
**issue:** I'm not sure to understand the intent behind this change. A scoped call is replaced with... a bound function! What changes is that now `resize()` is never called anymore, the only thing this lines does is to return a wrapper that binds the `resize()` function with a lexical this represented by the `arguments`... To me it looks like a dummy code.

The code would better be refactored with a removal of the deprecated `arguments` in favor of a spread operator, but in this end it would remain very similar, I'm not sure this will fix anything:
```js
                    resize(...args) {
                        if(typeof resize === 'function') {
                            resize.apply(this, args);
                        }
                    }
```

Reply:
Thanks for catch, refactoring is done, fix is moved in delegated function.
For reproducing you need to create narrow horizontal free form, and call initiation several times (I do it by focusing on file path field in properties).